container_test/handle_service_container_test.py

```diff
@@ -0,0 +1,132 @@
+import os
+import pytest
+import requests
+import time
+
+from typing import Optional
+from lib.AbstractHandle.AbstractHandleClient import AbstractHandle
+
+""" handle_service_container_test.py
+
+Very simple tests to ensure that local handle service server is functioning correctly.
+Requires the python libraries `pytest` and `requests` to be installed.
+
+Assumes that the handle_service2 is running locally on ports 8080.
+
+Use the wrapper shell script, `run_tests.sh`, to create the necessary set up and run the tests:
+
+sh scripts/run_tests.sh
+
+"""
+
+# setup TEST_TOKEN as env or provide a TOKEN_FILE_PATH
+# export TEST_TOKEN="your-kb-ci-token"
+TOKEN_FILE_PATH = None
+
+HANDLE_SERVICE_VERSION = "1.0.7"
+FILE_NAME = "mylittlefile"
+
+HS_URL = "http://localhost:8080"
+BLOB_URL = "https://ci.kbase.us/services/shock-api"
+WAIT_TIMES = [1, 2, 5, 10, 30]
+
+
+@pytest.fixture(scope="module")
+def ready():
+    wait_for_hs()
+
+    yield
+
+
+def wait_for_hs():
```

Comment:
The loop here is incorrect - on the last loop it'll print out that it's waiting for 30s, wait for 30s, and then immediately throw an error.

And yep, it's incorrect in the workspace too

Reply:
👍

Reply:
:+1: